brave/brave-core
# [WIP] Enable targeted Brave ads for Japan

Fixes https://github.com/brave/brave-browser/issues/4513

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:

- Confirm correct user model is loaded (search for `successfully loaded` in the console log, i.e. `Successfully loaded "en" user model for en_US locale`) for supported regions: United States of America, Canada, United Kingdom, France, Germany, Japan, Australia, New Zealand and Ireland
- Confirm page classification by searching for "tabClassification" in the console log
- Confirm Ads launch notification is shown and follows business logic for Japan, Australia, New Zealand and Ireland

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
